test/Microsoft.AspNetCore.Mvc.Core.Test/Internal/AuthorizationApplicationModelProviderTest.cs

```diff
@@ -116,6 +117,40 @@ public void CreateControllerModelAndActionModel_AllowAnonymousAttributeAddsAllow
         }
 
         [Fact]
+        public void CreateControllerModelAndActionModel_UseDefaultAuthorizationPolicyProvider()
```

Comment:
Additionally, this really seems like more than one thing is being tested - think about how many cases should exist, considering:
- default policy provider vs custom
- authorize data present or not 
- authoriza data on action or controller

These are the axes I can think of. Are they all important? Some of them?